inventories/test/routeros_hosts.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3
```

Comment:
a docstring right after this explaining why this exists and what it's for would help future developers (and me)

Reply:
Done/